### PR TITLE
Added Redfish Support

### PR DESCRIPTION
**Pull request by chris-lionetti**
Added many commands that support Redfish Servers, and now support subcomponents under System and Managers and Chassis
Added 'application/json' type to the restapi requests
tested against physical servers.

### CODE REVIEW

File: scripts/Drive.ps1 [hunk 1]
```diff
@@ -68,12 +68,18 @@ function Get-SwordFishDrive{
                            @{@odata.id=/redfish/v1/StorageServices/S1/Drives/1.4}...}
 .LINK
     https://redfish.dmtf.org/schemas/v1/Drive.v1_11_0.json
+    The Drives and Drives Collections will existing in the following Locations    
+        /redfish/v1/Drives/{DriveId}
```
Comment (rahlvers) [inline scripts/Drive.ps1]: This is not a valid URI (/redfish/v1/Drives)

File: scripts/Drive.ps1 [hunk 2]
```diff
@@ -68,12 +68,18 @@ function Get-SwordFishDrive{
...
+        /redfish/v1/Chassis/{ChassisId}/Drives/{DriveId}
+        /redfish/v1/Storage/{StorageId}/Drives/{DriveId}
```
Comment (rahlvers) [inline scripts/Drive.ps1:73]: Is this supposed to be a reference collection? If so, OK.   But drives only have two locations that are acceptable moving forward.  Most should be in /redfish/v1/Chassis/../Drives

Alternate is /redfish/v1/StorageServices/.../Drives

File: scripts/Drive.ps1 (hunk 1 above)
Comment (chris-lionetti) [reply scripts/Drive.ps1]: This has been fixed. See the latest update for this codebase. Reference to /redfish/v1/Drives/ have been removed.

File: scripts/Drive.ps1 (hunk 2 above)
Comment (chris-lionetti) [reply scripts/Drive.ps1:73]: Yes, indeed this is for a collection. The command can be directed to return collections, or the drives themselves, depending on the arguments used.

**Comment by rahlvers** [comment]
Approved in SSM TWG meeting. 4/12/22

**Review by rahlvers — APPROVED** [review]

**Pull request merged**